SoftLayer/CLI/virt/bandwidth.py

```diff
@@ -36,7 +36,14 @@ def cli(env, identifier, start_date, end_date, summary_period, quite_summary):
     """
     vsi = SoftLayer.VSManager(env.client)
     vsi_id = helpers.resolve_id(vsi.resolve_ids, identifier, 'VS')
-    data = vsi.get_bandwidth_data(vsi_id, start_date, end_date, None, summary_period)
+
+    # Summary period is broken for virtual guests, check VIRT-11733 for a resolution.
+    # For now, we are going to ignore summary_period and set it to the default the API imposes
+    if summary_period != 300:
+        click.secho("""The Summary Period option is currently set to the 300s as the backend API will throw an exception
+any other value. This should be resolved in the next version of the slcli.""", fg='yellow')
+    summary_period = 300
```

Comment:
I think is better changes on 
`@click.option('--summary_period', '-p', type=click.INT, default=300, show_default=True,  help="300, 600, 1800, 3600, 43200 or 86400 seconds")` 
and remove this line